addons/delivery/models/sale_order.py

```diff
@@ -88,6 +115,23 @@ def action_open_delivery_wizard(self):
             }
         }
 
+    def action_open_pickup_point_wizard(self):
+        if not self.partner_shipping_id.country_id:
```

Comment:
Only allow it if the carrier allow to use pickup point

Reply:
Alright, but isn't it enough to hide the button using this action when the carrier is not applicable?